### DISCUSSION TITLE
Search results not updated after removing text from search box

### DISCUSSION THREAD

**Discussion by supreet-bajaj** [microsoft-search/pnp-modern-search#3115] [General]
Dear team,

I am currently utilizing the search results web part along with a search box. While searching for something using the search box, the results are displayed correctly. However, when I remove the text from the search box, the search results do not update accordingly. Instead, it continues to display the same results based on the previous search item.

To resolve this issue, I have to refresh the page, at which point the correct results are finally shown. I kindly request your assistance in addressing this matter.

Thank you,
Supreet

**Comment by kasperbolarsen** [comment]
Are you using the X in the right hand side of the search box? The X is there for just that purpose.

**Comment by ccoley0473** [comment]
![Reset](https://github.com/microsoft-search/pnp-modern-search/assets/141053241/7c62d153-4b26-4b49-b71a-8bbd66d8ea12)

Edit your PnP Search box to reset on clear.

**Reply by supreet-bajaj** [reply]
Thanks for your reply. I have checked but can't see Reset query on clear option in my search box webpart. Attached screenshot is for reference.
Please let me know if I am missing something.

![image](https://github.com/microsoft-search/pnp-modern-search/assets/25117777/9557c62a-d45c-4f71-8f6e-853e014508df)

**Reply by wobba** [reply]
The option was added 11 months ago, so you might be using an older version of the web parts.
